spec/features/sign_in_out_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'spec_helper'
+
+describe 'sign in and out' do
+  it 'creates account, signs out, signs back in' do
+    email_address, personal_key = create_new_account
+
+    visit idp_logout_url
```

Comment:
creating the account signs you in. Need to sign out in order to test the sign in flow.